puppet/modules/quickstack/manifests/params.pp

```diff
@@ -58,9 +58,9 @@
   $cinder_dell_sc_san_login        = ['Admin'],
   $cinder_dell_sc_san_password     = ['CHANGEME'],
   $cinder_dell_sc_iscsi_ip_address = ['192.168.0.20'],
-  $cinder_dell_sc_iscsi_port       = [3260],
+  $cinder_dell_sc_iscsi_port       = ['3260'],
```

Comment:
A grep of the code shows these parameters are not used anywhere in quickstack, what is their purpose?